src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs

```diff
@@ -37,14 +37,14 @@ public TargetInfo(OSPlatform? os, Architecture? arch, Version targetFrameworkVer
 
```

Comment:
I think this file needs another patch:

```diff
diff --git a/src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs b/src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs
index 38f5306ea95..d1bba64388a 100644
--- a/src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs
+++ b/src/installer/managed/Microsoft.NET.HostModel/Bundle/TargetInfo.cs
@@ -33,21 +33,26 @@ public TargetInfo(OSPlatform? os, Architecture? arch, Version targetFrameworkVer
         {
             OS = os ?? HostOS;
             Arch = arch ?? RuntimeInformation.OSArchitecture;
-            FrameworkVersion = targetFrameworkVersion ?? net60;
+            FrameworkVersion = targetFrameworkVersion ?? new Version(7, 0);
 
             Debug.Assert(IsLinux || IsOSX || IsWindows);
 
-            if (FrameworkVersion.CompareTo(net70) >= 0)
+            if (FrameworkVersion.Major == 7)
             {
                 BundleMajorVersion = 7u;
                 DefaultOptions = BundleOptions.None;
             }
-            else if (FrameworkVersion.CompareTo(net60) >= 0)
+            else if (FrameworkVersion.Major == 6)
             {
                 BundleMajorVersion = 6u;
                 DefaultOptions = BundleOptions.None;
             }
-            else if (FrameworkVersion.Major == 3 && (FrameworkVersion.Minor == 0 || FrameworkVersion.Minor == 1))
+            else if (FrameworkVersion.Major == 5)
+            {
+                BundleMajorVersion = 2u;
+                DefaultOptions = BundleOptions.None;
+            }
+            else if (FrameworkVersion.Major == 3)
             {
                 BundleMajorVersion = 1u;
                 DefaultOptions = BundleOptions.BundleAllContent;
@@ -116,8 +121,6 @@ public override string ToString()
         public bool ShouldExclude(string relativePath) =>
             (FrameworkVersion.Major != 3) && (relativePath.Equals(HostFxr) || relativePath.Equals(HostPolicy));
 
-        private readonly Version net70 = new Version(7, 0);
-        private readonly Version net60 = new Version(6, 0);
         private string HostFxr => IsWindows ? "hostfxr.dll" : IsLinux ? "libhostfxr.so" : "libhostfxr.dylib";
         private string HostPolicy => IsWindows ? "hostpolicy.dll" : IsLinux ? "libhostpolicy.so" : "libhostpolicy.dylib";
     }
```

copy it in `/tmp/TargetInfo.cs.patch` and run `git apply /tmp/TargetInfo.cs.patch`

Reply:
I just copied this diff and made a file with this name TargetInfo.cs.patch and after that I applied it with git apply but I have this error:
`error: corrupt patch at line 44
`

<img width="636" alt="image" src="https://user-images.githubusercontent.com/68565441/177017602-77a78182-b070-422d-956c-9884c8b4123b.png">

Reply:
@am11 Oh I found the solution, I'm really happy because of learning these things